lib/watir-webdriver/attribute_helper.rb

```diff
@@ -74,7 +74,7 @@ def define_string_attribute(mname, aname)
 
     def define_boolean_attribute(mname, aname)
       define_method mname do
-        attribute_value(aname) == "true"
+        attribute_value(aname) =~ /(t|T)rue/
```

Comment:
The method should return boolean, while you changed it to return nil or fixnum. It's better to use one of:

```ruby
!!(attribute_value(aname) =~ /(t|T)rue/)
# or 
/(t|T)rue/ === attribute_value(aname)
```